src/include/binder/binder.h

```diff
@@ -286,6 +298,10 @@ class Binder {
     function::TableFunction* getScanFunction(
         common::FileType fileType, const common::ReaderConfig& config);
 
+    void extractExportData(
```

Comment:
instead change this to
```
ExportTableData getExportData(std::string selQuery, std::string tableName) 
```
so that the style is more consistent across this PR